bot/start.ts

```diff
@@ -207,9 +207,9 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
   bot.command('maintenance', superAdminMiddleware, async (ctx: MainContext): Promise<void> => {
     try {
       const [val] = await validateParams(ctx, 2, '\\<_on/off_\\>');
-      if (!val) return;
+      if (val === null) return;
       let config = await Config.findOne();
-      if (!config) {
+      if (config === null) {
```

Comment:
@Mersho are you absolutely sure that Config.findOne() can return null?

Reply:
yes

Reply:
@Mersho how about undefined?

Reply:
no, the type of config is `IConfig | null` .

Reply:
ok good